client/wildcard/src/components/Link/RouterLink/RouterLink.tsx

```diff
@@ -0,0 +1,23 @@
+import isAbsoluteUrl from 'is-absolute-url'
+import * as React from 'react'
+import { __RouterContext } from 'react-router'
+import { Link } from 'react-router-dom'
+
+import { AnchorLink, LinkProps } from '../AnchorLink'
+
+export function useInRouterContext(): boolean {
```

Comment:
It seems that we don't need `export` here.

```suggestion
function useInRouterContext(): boolean {
```